src/Uno.UWP/Storage/Pickers/FileOpenPicker.Android.cs

```diff
@@ -99,10 +99,53 @@ private string[] GetMimeTypes()
 				return new[] { "*/*" };
 			}
 
-			return FileTypeFilter
-				.Select(extension => MimeTypeService.GetFromExtension(extension))
-				.Distinct()
-				.ToArray();
+			List<string> mimeTypes = new List<string>();
+
+			Android.Webkit.MimeTypeMap? mimeTypeMap = Android.Webkit.MimeTypeMap.Singleton;
```

Comment:
Let's add a proper using here, remove the dispose below.

```suggestion
			using Android.Webkit.MimeTypeMap? mimeTypeMap = Android.Webkit.MimeTypeMap.Singleton;
```